test/routes/reportingInstructionsTest.js

```diff
@@ -26,12 +26,42 @@ const app = appSetup(createReportingInstructionRoute({
 }));
 
 describe('GET /reporting/:prisonNumber', () => {
+
+    afterEach(() => {
+        sandbox.reset();
+    });
+
     it('load html', () => {
         return request(app)
             .get('/1')
             .expect(200)
             .expect('Content-Type', /html/);
     });
+
+    it('calls getLicence from licenceService', () => {
+        return request(app)
+            .get('/1')
+            .expect(200)
+            .expect('Content-Type', /html/)
+            .expect(res => {
+                expect(licenceServiceStub.getLicence).to.be.calledOnce();
```

Comment:
What's the final result of going to this url? like what would the user see.. I'm not a big fan of peaking in to the implementation of controller if it can be helped.. I'd be tempted to cover this in an integration test?